.github/workflows/run_workflow.yml

```diff
@@ -0,0 +1,178 @@
+name: 'Azure e2e - Run Workflow'
+on:
+  schedule:
+    - cron: '0 16 * * *' # UTC 4pm, EST 11am, EDT 12pm
+  workflow_dispatch:
+    inputs:
+      target-branch:
+        description: 'Branch name of Cromwell repo to run tests on'
+        required: true
+        default: 'develop'
+        type: string
+      # Replace user data with seeded data provided by devOps (once available)
+      owner-subject:
+        description: 'Owner of billing project'
+        required: true
+        default: 'hermione.owner@quality.firecloud.org'
+        type: string
+      service-account:
+        description: 'Email address or unique identifier of the Google Cloud service account for which to generate credentials'
+        required: true
+        default: 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com'
+        type: string
+
+env:
+  BEE_NAME: '${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt}}-dev'
+  BROADBOT_TOKEN: '${{ secrets.BROADBOT_GITHUB_TOKEN }}' # github token for access to kick off a job in the private repo
+  RUN_NAME_SUFFIX: '${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}'
+
+jobs:
+  init-github-context:
+    runs-on: ubuntu-latest
+    outputs:
+      branch: ${{ steps.extract-inputs.outputs.branch }}
+      owner-subject: ${{ steps.extract-inputs.outputs.owner-subject }}
+      service-account: ${{ steps.extract-inputs.outputs.service-account }}
+    steps:
+#       NOTE: below was included in the rawls template but seems redundant due to defined values up on top
+#       Remove if it ends up being unecessary
+      - name: Get inputs or use defaults
```

Comment:
Just double checking that we think this is still necessary. It does seem redundant.